README.md

```diff
@@ -1,11 +1,202 @@
+
+The source for [Void Guides](https://guides.voidlinux.org/) is [void-linux/void-guides](https://github.com/void-linux/void-guides/) on GitHub.
+
+### Manual Pages (outside the scope of void-docs)
+
+The goal of a manual page is to document atomic components of an operating system. Executable programs, system calls, libraries, configuration files, and kernel routines are a few of the things likely to have a manual page.
+
+Void Linux follows the convention of most Linux and BSD variants and provides on-line reference manuals for most components of the operating system. See [man(1)](https://man.voidlinux.org/man) for details. 
+
+There is no single source for manual pages, but a good place to look is within the project they document. For example, to contribute to the manual documentation for [openssl(1)](https://man.voidlinux.org/openssl), you engage the OpenSSL project to propose changes to the [source for the openssl(1) man pages](https://github.com/openssl/openssl/tree/master/doc/man1).
+
+## Style Guide
+
+This style guide outlines the standards for contributing to the
+[void-docs](https://github.com/void-linux/void-docs/) project. The Handbook is generated from an
+[mdBook](https://rust-lang-nursery.github.io/mdBook/) stored in the
+[void-docs](https://github.com/void-linux/void-docs/) repository.
+
+### Formatting
+
+For markdown formatting, the
+[void-docs](https://github.com/void-linux/void-docs/) project uses the
+[Versioned Markdown](https://github.com/bobertlo/vmd) format, and enforces use
+of the auto-formatter `vmdfmt`, which works very similarly to `gofmt`. Most
+valid markdown is accepted by the formatter. The output format is described in
+the project's [README](https://github.com/bobertlo/vmd/blob/master/README.md).
+
+Void provides the package `vmdfmt`. Otherwise you may `go get` from the repo:
+
+```
+$ go get https://github.com/bobertlo/vmd/cmd/vmdfmt
+```
+
+To format a file you have edited, run:
+
+```
+vmdfmt -w <filepath>
+```
+
+To format the entire *mdbook* from the repository root, outputting a list of
+files modified, run:
+
+```
+vmdfmt -w -l <filepath>
+```
+
+### Commands
+
+Command code-blocks should start with a `#` or `$` character, indicating whether
+the command should be run as `root` or a regular user, respectively.
+
+For example:
+
+```
+# vi /etc/fstab
+```
+
+And not:
+
+```
+$ sudo vi /etc/fstab
+```
+
+And also not:
+
+```
+vi /etc/fstab
+```
+
+Command code-blocks should not be used to describe routine tasks documented
+elsewhere in this Handbook. For example, when writing documentation for the
+`foo` package, do not provide a command code-block stating that one should
+install it via `xbps-install foo`.
```

Comment:
Most of this part is already in the `contributing/void-docs` section, iirc. So we can try to port what's missing to that instead.